liquid/doc.go

```diff
@@ -146,6 +146,10 @@ package liquid
 // This type is used to distinguish project UUIDs from other types of string values in structs and function signatures.
 type ProjectUUID string
 
+// ProjectCommitmentUUID identifies a project commitment within a liquid.
+// This type is used to distinguish commitment UUIDs from other types of string values in structs and function signatures.
+type ProjectCommitmentUUID string
```

Comment:
This is clearly named after the `project_commitments` table in Limes, but that is an implementation detail that we do not need to expose here.

```suggestion
type CommitmentUUID string
```

Same reasoning as why the type in LIQUID is called `Commitment` even though in Limes it's `ProjectCommitment`.